proc/groupby.go

```diff
@@ -15,12 +15,13 @@ import (
 	"github.com/brimsec/zq/zcode"
 	"github.com/brimsec/zq/zng"
 	"github.com/brimsec/zq/zng/resolver"
+	"github.com/pkg/errors"
```

Comment:
Was this intentional, or did you mean just `"errors"`?

Reply:
oops, it was certainly not! i should pay closer attention to what goimports is doing for me :)

Reply:
I figured, the same has happened to me :)